docs/developer-docs/integrations/sns/managing/making-proposals.md

```diff
@@ -443,6 +444,52 @@ Then the resulting metadata will end like this where only the `description` fiel
     description : "UPDATED Sample SNS used for educational purposes";
 ```
 
+### `MintSnsTokens`
+
+Each SNS can have SNS tokens in its treasury, but it also has the ability to mint new SNS tokens to a particular user. This is a ["critical" proposal type](#criticalproposals), which means it is subject to higher voting thresholds before it is accepted.
```

Comment:
I would copy the "this is a critical proposal" part to the other proposals for which this is true (treasury and deregister canister)

Reply:
Of course, done!